API.md

```diff
@@ -2310,7 +2310,7 @@ following options:
           clean up the files generated by the framework. This can be done by keeping track
           of which files are used (e.g. using the `request.app` object), and listening to
           the server `'response'` event to perform any needed cleanup.
-    - `parse` - can be `true`, `false`, or `gunzip`; determines if the incoming payload is
+    - `parse` - can be `true`, `false`, `gunzip`, or `br`; determines if the incoming payload is
```

Comment:
@nlf it seems that this will be added as a plugin but in any case I updated the API doc (not sure if the added text is the best / most comprehensible though).

Regarding the use of `br` (instead of `brotli`), it is the naming scheme that it is being used: https://github.com/google/brotli/issues/299